src/Sparrow.Server/Json/Sync/JsonOperationContextSyncApiExtensions.cs

```diff
@@ -35,25 +36,23 @@ public static unsafe BlittableJsonReaderObject ReadForMemory(this JsonOperationC
             // todo: maybe use ManagedPinnedBuffer here
             var maxByteSize = Encodings.Utf8.GetMaxByteCount(jsonString.Length);
 
-            fixed (char* val = jsonString)
+            var buffer = ArrayPool<byte>.Shared.Rent(maxByteSize);
+            try
             {
-                var buffer = ArrayPool<byte>.Shared.Rent(maxByteSize);
-                try
+                // PERF: There is no advantage to fix the array, since internally the same will happen.
+                // If the framework does indeed improves the implementation to work natively, we will
+                // miss it because of it. 
+                https://issues.hibernatingrhinos.com/issue/RavenDB-20321
```

Comment:
Please check the warning

Reply:
WOW, that compiles!!!

Reply:
Yeah, that is a common issue.
It's a label named `http:` followed by a comment :-)